library-best-practices/JLBP-2.md

```diff
@@ -9,12 +9,12 @@
     longer than your dependencies. Historically this was a
     big problem for Guava, which made a major version bump
     every 6 months, although Guava no longer does this.
-- Use package-protected packages/classes when the API is internal
+- Use package-protected classes and methods when the API is internal
```

Comment:
I believe "package-protected packages" was typo, because 

> A package is always accessible.
https://docs.oracle.com/javase/specs/jls/se8/html/jls-6.html#jls-6.6.1